core-web/libs/portlets/edit-ema/portlet/src/lib/utils/index.ts

```diff
@@ -14,6 +14,10 @@ export function insertContentletInContainer(action: ActionPayload): {
     pageContainers: PageContainer[];
     didInsert: boolean;
 } {
+    if (!action) {
+        return { pageContainers: [], didInsert: false };
+    }
```

Comment:
Where I can call it without action? and if so... why? do I need to show something to the user?

Reply:
This is to check the empty response in `getFormId`

Reply:
I reverted this.